languages/common-lisp/exercises/concept/lillys-lasagna-leftovers/.docs/introduction.md

```diff
@@ -0,0 +1,56 @@
+In Common Lisp a functions argument list (also known as a ('lambda list')[lambda-list]) can have arguments of different types. These different types are designated with the use of ("lambda list keywords")[lambda-list-keyword] which all begin with `&`. The most commonly used types are optional, keyword and rest arguments types. Every parameter in the lambda list after a particular lambda list keyword is will be of that type. A lambda list keyword can only be used once in a lambda list.
```

Comment:
This is great to have here (the explanation of the lambda-list terminology)! It should probably also be in a concept file somewhere if it's not already :slightly_smiling_face:

Reply:
I wonder if lambda list should be its own concept? (it won't have its own exercises... but maybe it is enough of a topic on its own?)

Reply:
I'm not sure how you've addressed this just yet, but I wouldn't be opposed to that. It might be nice, as it comes up again in macros and destructuring!